flink-table/flink-table-planner-blink/pom.xml

```diff
@@ -149,20 +138,30 @@ under the License.
 			<version>${calcite.version}</version>
 			<exclusions>
 				<!--
-				"mvn dependency:tree" as of Calcite 1.21:
-
-				[INFO] +- org.apache.calcite:calcite-core:jar:1.21.0:compile
-				[INFO] |  +- commons-codec:commons-codec:jar:1.10:compile
-				[INFO] |  +- org.apache.calcite:calcite-linq4j:jar:1.21.0:compile
-				[INFO] |  +- org.apache.commons:commons-lang3:jar:3.3.2:compile
-				[INFO] |  +- com.fasterxml.jackson.core:jackson-core:jar:2.10.1:compile
-				[INFO] |  +- com.fasterxml.jackson.core:jackson-annotations:jar:2.10.1:compile
-				[INFO] |  +- com.fasterxml.jackson.core:jackson-databind:jar:2.10.1:compile
-				[INFO] |  +- com.google.guava:guava:jar:19.0:compile
-				[INFO] |  \- com.jayway.jsonpath:json-path:jar:2.4.0:compile
+				"mvn dependency:tree" as of Calcite 1.22.0:
+
+				[INFO] +- org.apache.calcite:calcite-core:jar:1.22.0:compile
+                [INFO] |  +- org.apache.calcite:calcite-linq4j:jar:1.22.0:compile
+                [INFO] |  +- com.fasterxml.jackson.core:jackson-annotations:jar:2.10.1:compile
+                [INFO] |  +- org.apache.calcite.avatica:avatica-core:jar:1.16.0:compile
```

Comment:
I execute `mvn dependency:tree` in flink-table-planner-blink path, I do not find `avatica-core`

+- org.apache.calcite:calcite-core:jar:1.22.0:compile
[INFO] |  +- org.apache.calcite:calcite-linq4j:jar:1.22.0:compile
[INFO] |  +- com.fasterxml.jackson.core:jackson-annotations:jar:2.10.1:compile
[INFO] |  +- org.apiguardian:apiguardian-api:jar:1.1.0:compile
[INFO] |  +- com.fasterxml.jackson.core:jackson-core:jar:2.10.1:runtime
[INFO] |  +- com.fasterxml.jackson.core:jackson-databind:jar:2.10.1:runtime
[INFO] |  +- com.jayway.jsonpath:json-path:jar:2.4.0:runtime
[INFO] |  |  \- net.minidev:json-smart:jar:2.3:runtime
[INFO] |  |     \- net.minidev:accessors-smart:jar:1.2:runtime
[INFO] |  |        \- org.ow2.asm:asm:jar:5.0.4:runtime
[INFO] |  +- commons-codec:commons-codec:jar:1.10:runtime
[INFO] |  +- org.apache.commons:commons-lang3:jar:3.3.2:compile
[INFO] |  \- commons-io:commons-io:jar:2.4:compile

Reply:
Yes, you are right, thanks for the check !

Reply:
We should only keep `compile` dependency here

Reply:
I agree, thanks.